src/styles/withTheme.js

```diff
@@ -0,0 +1,33 @@
+// @flow weak
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import wrapDisplayName from 'recompose/wrapDisplayName';
```

Comment:
What about using the `withStyles`  helper as inspiration? https://github.com/callemall/material-ui/blob/next/src/styles/withStyles.js#L8